SDWebImage/SDWebImageManager.h

```diff
@@ -94,7 +94,11 @@ typedef NS_OPTIONS(NSUInteger, SDWebImageOptions) {
      * images to a size compatible with the constrained memory of devices.
      * If `SDWebImageProgressiveDownload` flag is set the scale down is deactivated.
      */
-    SDWebImageScaleDownLargeImages = 1 << 12
+    SDWebImageScaleDownLargeImages = 1 << 12,
+    /**
+     * Use SDAnimatedImage instead of UIImage(NSImage) instance if needed to enhance animated image rendering on SDAnimatedImageView.
+     */
+    SDWebImageUseSDAnimatedImage = 1 << 13,
```

Comment:
I feel that this option is kind of needless. Why wouldn't one want to use SDAnimatedImage were it better than UIImage in many ways 😉

Reply:
@skyline75489 I think we should not replace all `UIImageView` and it's configuatable. This `SDAnimatedImageView` is used for high-performance banlanced of CPU & memory. But for some small animated images (especially small UI icon), UIImage's build-in one is the best choice. So we need this :)

And also, some build-in class can't be replaced with that image view. For example, UIButton's `setImage:forState:` can only show animated images with `UIAnimatedImage`. So this case our solution will fail.